src/brouwers/kitreviews/forms.py

```diff
@@ -26,8 +26,8 @@ class Meta:
         model = KitReview
         exclude = ('model_kit', 'html', 'reviewer')
 
-    def __init__(self, user, *args, **kwargs):
+    """def __init__(self, user, *args, **kwargs):
```

Comment:
this is needed - we must restrict the possible album choices to the albums owned by the passed in user. Otherwise they can see that certain private albums exist.

the correct queryset for this is `albums = user.album_set.filter(trash=False)` I think